src/main/java/io/strimzi/kafka/bridge/amqp/converter/AmqpDefaultMessageConverter.java

```diff
@@ -139,7 +139,7 @@ public Message toMessage(String address, KafkaConsumerRecord<String, byte[]> rec
 	}
 
 	@Override
-	public List<KafkaProducerRecord<String, byte[]>> toKafkaRecords(String kafkaTopic, Collection<Message> messages, Integer partition) {
+	public List<KafkaProducerRecord<String, byte[]>> toKafkaRecords(String kafkaTopic, Integer partition, Collection<Message> messages) {
```

Comment:
different `partition` parameter name from the other `toKafkaRecord`. I would use just `partition` here.